tests/qos/qos_sai_base.py

```diff
@@ -8,9 +8,10 @@
 from tests.common.helpers.assertions import pytest_assert, pytest_require
 from tests.common.mellanox_data import is_mellanox_device as isMellanoxDevice
 from tests.common.dualtor.dual_tor_utils import upper_tor_host,lower_tor_host,dualtor_ports
-from tests.common.dualtor.mux_simulator_control import mux_server_url, toggle_all_simulator_ports
+from tests.common.dualtor.mux_simulator_control import toggle_all_simulator_ports, get_mux_status, check_mux_status, validate_check_result
```

Comment:
@XuChen-MSFT There is a new unused import alert, we can add "# lgtm[py/unused-import]" after this line to ignore it.